pkg/telemetry/telemetry.go

```diff
@@ -60,7 +61,7 @@ func (t *Telemetry) Initialize(stepName string) {
 
 	t.baseData = BaseData{
 		Orchestrator:    t.provider.OrchestratorType(),
-		TemplateName:    os.Getenv("PIPER_PIPELINE_TEMPLATE_NAME"),
+		TemplateName:    piperutils.StringWithDefault(os.Getenv("PIPER_PIPELINE_TEMPLATE_NAME"), "custom"),
```

Comment:
We could also use `unknown`/ `n/a`, maybe in the future we can determine exactly if it's custom.